### PR TITLE
Update External JWT Provider Doc

### PR DESCRIPTION
**Pull request by jobannon**
# Summary 
- Minor changes to documentation and spelling.
- Update lanaguage around managed domains.  A managed domain is not a requirement.

### CODE REVIEW

File: site/docs/v1/tech/identity-providers/external-jwt/index.adoc [hunk 1]
```diff
@@ -166,20 +174,20 @@ Optionally a single key may be designated as the default key. You may use this f
 +
 In this scenario, there will be no `kid` to indicate which key was used to sign the JWT, so if a default key has been designated, the default key will be used to verify the signature.
 +
-If you will not have JWTs that do not contain the `kid` or other key identifier, you will not use this feature. When this toggle is enabled, the `Key identifier` field will be disabled and no longer required.
+If you will not have JWTs that do not contain the `kid` or another key identifier, you will not use this feature. When this toggle is enabled, the `Key identifier` field will be disabled and no longer required.
```
Comment (mooreds) [inline site/docs/v1/tech/identity-providers/external-jwt/index.adoc:177]: I think this means:

"If you only have JWTs with the `kid` claim, or another key identifier, you will not use this feature. When this toggle is enabled, the [field]#Key identifier# field will be disabled and no longer required."

File: site/docs/v1/tech/identity-providers/external-jwt/index.adoc [hunk 2]
```diff
@@ -8,17 +8,24 @@ description: Complete a FusionAuth login with an external JWT from a third party
...
 
 - link:/docs/v1/tech/apis/identity-providers/external-jwt/[External JWT API]
 - link:/docs/v1/tech/apis/jwt#reconcile-a-jwt[Reconcile a JWT API]
 
 Using this identity provider allows you to authenticate with a third party that can provide you a JSON Web Token (JWT). Then you may take this JWT and send it to FusionAuth to reconcile the user. The reconcile process is essentially logging that user into a particular FusionAuth application.
 
-The configuration for this identity provider allows you to configure some claim mapping from the third party JWT to the FusionAuth user, and it will require one or more keys that will allow FusionAuth to verify the signature of the third party JWT. These keys will be added on the `Verification Keys` section documented below.
+The configuration for this identity provider allows you to configure some claim mapping from the third party JWT to the FusionAuth user, and it will require one or more keys that will allow FusionAuth to verify the signature of the third party JWT. These keys will be added to the `Verification Keys` section documented below.
 
 {empty} +
```
Comment (mooreds) [inline site/docs/v1/tech/identity-providers/external-jwt/index.adoc:20]: This leaves weird empty space. Can we get rid of it?

File: site/docs/v1/tech/identity-providers/external-jwt/index.adoc [hunk 3]
```diff
@@ -8,17 +8,24 @@ description: Complete a FusionAuth login with an external JWT from a third party
...
 
+====
```
Comment (mooreds) [inline site/docs/v1/tech/identity-providers/external-jwt/index.adoc:22]: All the other notes I've seen look like

```
[NOTE]
====
Note contents
====
```

Please maintain that pattern.

File: site/docs/v1/tech/identity-providers/external-jwt/index.adoc [hunk 4]
```diff
@@ -44,12 +51,12 @@ image::identity-providers/identity-provider-external-jwt-add.png[Add External JW
...
 
 [field]#Name# [required]#Required#::
-A unique name to identity the identity provider. This name is for display purposes only and it can be modified later if desired.
+A unique name to identify the identity provider. This name is for display purposes only and it can be modified later if desired.
 
 [field]#Reconcile lambda# [optional]#Optional# [since]#Available since 1.17.0#::
 A lambda may be utilized to map custom claims from the JWT. This may be used in addition to, or in place of the claim mapping on the JWT tab.
 +
-To configure a lambda, navigate to [breadcrumb]#Settings -> Lambdas#.
+To configure a lambda, navigate to [breadcrumb]#Customizations -> Lambdas#.
```
Comment (mooreds) [inline site/docs/v1/tech/identity-providers/external-jwt/index.adoc:59]: Great catch!

File: site/docs/v1/tech/identity-providers/external-jwt/index.adoc [hunk 5]
```diff
@@ -120,9 +127,10 @@ For example, if you select `company` as the incoming claim, and specify `Registr
 
 == Managed Domains
 
-In order to successfully reconcile a JWT to FusionAuth using this identify provider, you will need to specify one more managed domains. A managed domain will indicate to FusionAuth that this identify provider is able to reconcile a user based upon their email address.
+To successfully reconcile a JWT to FusionAuth using this identity provider, you can *optionally* specify one more managed domains.
+A managed domain will indicate to FusionAuth this identity provider can reconcile a user based upon their email address.
 
-For example if a user's email address is `jared@piedpiper.com`, in order for a JWT owned by Jared to be reconciled to FusionAuth using this identity provider, `piedpiper.com` will need to be configured as a managed domain.
+For example, if a user's email address is `jared@piedpiper.com`, in order for a JWT owned by Jared to be reconciled to FusionAuth using this identity provider, `piedpiper.com` could be configured as a managed domain.
```
Comment (mooreds) [inline site/docs/v1/tech/identity-providers/external-jwt/index.adoc:133]: Can you add to the end: "You may leave this blank. If you do so, then any email can be used with this identity provider. This is helpful if you are not certain which email domains your users will have."

**Review by mooreds — CHANGES_REQUESTED** [review]
A few small changes, other than that looks good.

**Comment by mooreds** [comment]
@jobannon how is this going?

**Comment by mooreds** [comment]
@jobannon ping!

**Comment by jobannon** [comment]
@mooreds  pong! 😉

**Pull request merged**